libs/s25main/gameData/DoorConsts.h

```diff
@@ -12,17 +12,17 @@
 /// Y-value for the point where the carrier disappears inside the building for each nation and building
 constexpr helpers::MultiEnumArray<int8_t, Nation, BuildingType> DOOR_CONSTS = {
   {// Nubier
-   {5, 10, 13, 0,  10, 0,  0,  0, 0,  6,  8, 8,  8,  8, 6,  0, 10, 10, 12, 14,
-    9, 5,  11, 19, 19, 12, 18, 0, -6, 19, 0, 12, 11, 6, 10, 0, 0,  -1, 4,  13},
+   {5, 10, 13, 0,  10, 0,  11, 19, 0,  6,  8, 8,  8,  8, 6,  0, 10, 10, 12, 14,
```

Comment:
The nubian vinyard doesn't need an offset?

Reply:
Yes the nubian vinyard has no offset

Reply:
Ok thanks.

Please don't resolve conversations you answered to unless it is just applying a suggested fix from the comment.  
That makes it much easier to keep track of things I already looked at and what are new answers. The person starting the conversation should resolve it when the topic of the discussion is finished. E.g. when the original question wasn't fully answered or the reviewer has a follow-up question which the other person might not be aware of.